DESCRIPTION

```diff
@@ -60,7 +60,9 @@ Suggests:
     survMisc,
     survRM2,
     testthat,
-    tidyr
+    tidyr,
+    doRNG,
+    tictoc
```

Comment:
How informative are the timings in the vignettes? Is it worth adding another dependency to be installed during each CI run?

If the simulations are small, and they are running on minimal hardware like a CRAN or GitHub machine, it's not clear to me if this will be that informative to an end user that will want to run many simulations on more robust hardware.